engine/src/main/java/io/camunda/zeebe/engine/processing/streamprocessor/writers/LegacyTypedStreamWriter.java

```diff
@@ -14,7 +14,13 @@
 
 /** Things that only a stream processor should write to the log stream (+ commands) */
 public interface LegacyTypedStreamWriter
-    extends LegacyTypedCommandWriter, TypedEventWriter, TypedRejectionWriter {
+    extends TypedCommandWriter, TypedEventWriter, TypedRejectionWriter {
```

Comment:
:question: sorry I'm slightly confused now between the legacy and the without legacy interfaces :D what is the difference?

Reply:
Legacy interface has `reset() + flush()`. The new ones don't